src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1008,7 +1008,13 @@ class RPCParser
                 jvRequest[jss::binary] = true;
         }
 
-        jvRequest["transaction"]    = jvParams[0u].asString ();
+        if (jvParams.size () > 3)
+        {
```

Comment:
I'm not sure this parses right. As written, if we specify a range we must specify binary. And something like `tx 100 200` would be misparsed an no error would be issued (at least by my reading of the code). I'd love to change this so we'd give an error if we can't parse correctly and we correctly make binary an optional parameter. We probably also want to give errors for too large a range or for min>max.

Reply:
Done